src/olympia/lib/log_settings_base.py

```diff
@@ -22,24 +22,12 @@
         '()': olympia.core.logger.Formatter,
         'datefmt': '%H:%M:%S',
         'format': ('%s %s: [%%(USERNAME)s][%%(REMOTE_ADDR)s] %s'
-                   % (settings.HOSTNAME, settings.SYSLOG_TAG, error_fmt)),
+                   % (settings.HOSTNAME, settings.MOZLOG_NAME, error_fmt)),
```

Comment:
'error' formatter doesn't appear to be explicitly referenced anywhere, but I was nervous about removing it in case it was used as a fallback